### ISSUE TITLE
update page with specific permission requirements for virtual tables

### ISSUE THREAD

**Issue by FintanGuihen** [MicrosoftDocs/powerapps-docs#4616]
After weeks of working with Microsoft support to identify why users could not use virtual tables, they have finally advised that

"only system administrators and system customizers possess the necessary permissions to access this virtual table. It has come to our attention that users with custom roles, basic privileges, and similar permissions are unable to view or edit the said table. "

These permissions are much too high for standard users, and so virtual tables are useless to many people in this form.

can the required permissions please be added to the document as it  may save a lot of wasted time for other Microsoft customers.

Thanks,

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 42ae2cb6-ce2a-21c7-cb81-b60e7b026753
* Version Independent ID: 5a3cb7c0-6881-1d96-0dc3-639324a4568c
* Content: [Get started with virtual tables (entities) (Microsoft Dataverse) - Power Apps](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/virtual-entities/get-started-ve)
* Content Source: [powerapps-docs/developer/data-platform/virtual-entities/get-started-ve.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/developer/data-platform/virtual-entities/get-started-ve.md)
* Service: **powerapps**
* Sub-service: **dataverse-developer**
* GitHub Login: @NHelgren
* Microsoft Alias: **nhelgren**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @FintanGuihen. This issue has been assigned to the doc author for review.

**Comment by phecke** [comment]
@FintanGuihen I do not believe that such a restriction of permission applies to using virtual tables (VT). You must be system admin or customizer to create a VT. I will contact the PM for clarification.

**Comment by FintanGuihen** [comment]
Hi Peter.

This information came from Microsoft support.
I raised a ticket after I was unable to grant users the ability to use a
power app that uses this table.

I can provide the ticket ID if needed.

Thanks,
Fintan.

**Comment by phecke** [comment]
@FintanGuihen A VT implementation requires writing plug-ins if you are implementing a custom data provider. To register plug-ins with Dataverse you need the System Admin or System Customizer role. However, for end-users to use the VT should only require the appropriate permissions to access the data on the remote system through the VT. I am trying to get in touch with the PM for clarification.

In this doc [Limitations-of-virtual-tables](https://learn.microsoft.com/power-apps/developer/data-platform/virtual-entities/get-started-ve#limitations-of-virtual-tables) it states: _Access to the virtual table data can be turned on or off for individual users based on their security role.

In summary, you need System Admin or Customizer role to register plug-ins with Dataverse for the CRUD operations of a custom data provider. End-users need to be authorized individually for read/write access to the VT.
